tests/integration/examples_test.py

```diff
@@ -134,3 +134,12 @@ def test_add_remove_team_with_slaves(eth1_up, eth2_up):
         assertlib.assert_state_match(desired_state)
 
     assertlib.assert_absent("team0")
+
+
+@pytest.mark.skipif(
+    os.environ.get("CI") == "true",
```

Comment:
Please use this line like you did in `sriov_test.py`:

```python
@pytest.mark.xfail(
    raises=NmstateNotSupportedError,
    reason="The device does not support SR-IOV.",
)
```

Reply:
I've added the xfail after the skipif. The skipif is needed because the xfail takes too much time for CI. Thanks!